config/operator/rbac/operator_editor_role.yaml

```diff
@@ -0,0 +1,32 @@
+# permissions for end users to edit operators.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
the `clusterlink.net` objects are **namespaced**, thus I think we should have a Role and RoleBinding not cluster scoped roles and bindings.

Reply:
I will change it in the next phase, when we have the new controller